packages/node_modules/@webex/plugin-meetings/src/util/webex-meetings-error.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @extends Error
+ * @property {number} code - Error code
+ */
+class WebexError extends Error {
```

Comment:
Would like this in the `common/errors` folder as well.

Reply:
fixed